server/controllers/profile.js

```diff
@@ -1,97 +1,110 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
-const Profile = require('../models/Profile');
+const Profile = require("../models/Profile");
 
 const asyncHandler = require("express-async-handler");
 
-// @route POST 
+// @route POST /profile/new
 // Create New User Profile
 exports.newUser = asyncHandler(async (req, res) => {

```

Comment:
maybe we can change the name of the controller to `createProfile` or `newProfile` for increased semantics as profile and user are individual models.